src/cls/rgw/cls_rgw_ops.h

```diff
@@ -1002,56 +1002,80 @@ struct cls_rgw_lc_get_next_entry_op {
 WRITE_CLASS_ENCODER(cls_rgw_lc_get_next_entry_op)
 
 struct cls_rgw_lc_get_next_entry_ret {
-  pair<string, int> entry;
+  cls_rgw_lc_entry entry;
 
   cls_rgw_lc_get_next_entry_ret() {}
 
   void encode(bufferlist& bl) const {
-    ENCODE_START(1, 1, bl);
+    ENCODE_START(2, 2, bl);
     ::encode(entry, bl);
     ENCODE_FINISH(bl);
   }
 
   void decode(bufferlist::iterator& bl) {
-    DECODE_START(1, bl);
-    ::decode(entry, bl);
+    DECODE_START(2, bl);
+    if (struct_v <= 1) {
+      pair<string, int> entry_pair;
+      ::decode(entry_pair, bl);
+      entry.bucket = entry_pair.first;
+      entry.status = entry_pair.second;
+    }
```

Comment:
Actually, you have changed all the definition of `pair<string, int> entry` to `cls_rgw_lc_entry entry`, so I think this condition will never appear. Maybe we can discard the code above. What do you think? @yehudasa @yaozongyou